src/Containers/App/App.scss

```diff
@@ -0,0 +1,67 @@
+*,
```

Comment:
The styling looks really great. I like the design and how the top header is fixed so that everything flows behind it nicely.